Gemfile

```diff
@@ -43,7 +43,7 @@ gem 'attr_encrypted', '3.1.0'
 gem 'aws-sdk-s3', '~> 1'
 gem 'aws-sdk-sns', '~> 1'
 gem 'betamocks', git: 'https://github.com/department-of-veterans-affairs/betamocks', branch: 'master'
-gem 'bgs_ext', git: 'https://github.com/department-of-veterans-affairs/bgs-ext.git', require: 'bgs'
+gem 'bgs_ext', git: 'https://github.com/department-of-veterans-affairs/bgs-ext.git', require: 'bgs', branch: 'slakrhakr/API-3272'
```

Comment:
Requires this pr for bgs-ext before this can go through
https://github.com/department-of-veterans-affairs/bgs-ext/pull/51

Reply:
done